elastic/kibana
# [Response Ops][Alerting] Alert preview index mappings should be allow-listed for update

Resolves https://github.com/elastic/kibana/issues/189211

## Summary

Adds alert preview index prefix to the list of valid prefixes so that alert preview index mappings are correctly updated on upgrade.

## To Verify

1. Start an older version of Kibana (I used 8.12) with `yarn es snapshot --license trial --ssl -E path.data=../test_189211` and `yarn start --ssl`
2. Create a detection rule and run the rule preview before creating the rule. Go to Index Management and you should see a preview index in the list `.internal.preview.alerts-security.alerts-default-000001`
3. Using the same ES data, run Kibana on `main`. Preview a detection rule. You should see the following the logs:
```
Found unexpected concrete index name \".internal.preview.alerts-security.alerts-service-desk-000007\" while expecting index with one of the following prefixes: [.ds-.alerts-,.internal.alerts-,.alerts-] Not updating mappings or settings for this index.
```
4. Using the same ES data, run Kibana on this branch. Preview a detection rule. You should no longer see the above log message.
